ruby/alias_manager.rb

```diff
@@ -0,0 +1,103 @@
+# Alias Manager Pseudocode
+
+# 1) Prompt user for real_name_str input
+# 2) Call real_name_str.downcase! method to store all characters as lowercase
+# 3) Call real_name_str.split and store in real_name_arr
+# 4) Identify all vowels in array
+# 5) Change each vowel in alias to next vowel (a,e,i,o,u) and store in same position
+# in alias_arr using method next_vowel
+# 5a) Be sure to include special case of "u" conversion to "a"
+# 6) Identify all consonants in array
+# 7) Change each consonant in alias to next consonant (a,b,c,d,f,g,h,j...) and store 
+# in same position in alias_arr using method next_vowel
+# 7a) Be sure to include special case of "z" conversion to "b"
+# 8) Identify all non-alphabet characters and store without changing in same position
+# in alias_arr
+# 9) Call Array#join with " " parameter to convert alias_arr to array of 2 strings
+# 10) Swap positions of two strings 
+# 11) Call Array#join to combine the two elements in a single string with space in 
+# between and store in alias_str
+# 12) Print out alias_str to console
+# 13) Create aliases hash to store each alias_str as key and real_name_str as value
+
+
+
+def next_vowel(letter)
+	index = VOWELS.index(letter)
```

Comment:
Don't use constants in your code at this time.  Vowels does make sense in this context because it will not change but constants can make it difficult to track your variables in many cases.

For the time being just reuse the string `"aeiou"`